spec/system/validate_collection_deposit_spec.rb

```diff
@@ -79,11 +79,8 @@
     find('.nav-link', text: 'Save your collection').click
     expect(page).to have_css('.nav-link.active', text: 'Save your collection')
     click_link_or_button('Save', class: 'btn-primary')
-    expect(page).to have_css('h1', text: collection_title_fixture)
-    expect(page).to have_current_path(collection_path)
-
-    # No Alert!
-    expect(page).to have_no_css('.alert-danger', text: 'Required fields have not been filled out.')
```

Comment:
This test failed consistently for me. And it turns out these expectations... are not at all what we expect. They only passed prior because the tests ran too fast. What we *actually* expect is the page to be the collection wait page which is what we have below.

Turning on capybara-lockstep can cause tests that used to pass to fail for this reason, so this is not unexpected.